addOns/requester/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 
 
+## Added 
+- Added Button to LowerCase Headers Name
```

Comment:
The blanks at lines 8 and 9 can be removed and just one after the note. The note should be a sentence (add a period, fix the caps, make name plural) and you should add an issue reference.